Dockerfile

```diff
@@ -1,49 +1,47 @@
-# TODO: we should make use of BUILDPLATFORM and TARGETPLATFORM to figure out cross compiling
-#  as mentioned here: docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide
-#  then we could use the host architecture to simply compile to the target architecture without
-#  emulating the target architecture (thereby making the build hyper slow). the general gist is
-#  we add arm (or whatever architecture) repositories to apt and then install our dependencies
-#  with the architecture suffix, eg. :arm64. then we just need to set a bunch of cmake variables
-#  probably with the use of a cmake toolchain file so that cmake can make sure to use the
-#  binaries that can target the target architecture. from there bob is your uncle maybe..
-
-####################################################################
 FROM ubuntu:24.04 AS builder
 LABEL org.opencontainers.image.authors="Kevin Kreiser <kevinkreiser@gmail.com>"
 
-ARG CONCURRENCY
-ARG ADDITIONAL_TARGETS
-ARG VERSION_MODIFIER=""
-
 # set paths
 ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
 ENV LD_LIBRARY_PATH=/usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
 RUN export DEBIAN_FRONTEND=noninteractive && apt update && apt install -y sudo
 
 # install deps
-WORKDIR /usr/local/src/valhalla
-COPY ./scripts/install-linux-deps.sh /usr/local/src/valhalla/scripts/install-linux-deps.sh
-RUN bash /usr/local/src/valhalla/scripts/install-linux-deps.sh
-RUN rm -rf /var/lib/apt/lists/*
+WORKDIR /src/valhalla
+COPY scripts/install-linux-deps.sh scripts/install-linux-deps.sh
+RUN bash ./scripts/install-linux-deps.sh
 
-# get the code into the right place and prepare to build it
-ADD . .
-RUN ls -la
-RUN git submodule sync && git submodule update --init --recursive
```

Comment:
we don't carry over the .git folder anymore, we assume the host checks out the right stuff